packages/backend/src/graphql/schema.graphql

```diff
@@ -770,6 +773,41 @@ type PaymentPointerWithdrawal {
   paymentPointer: PaymentPointer!
 }
 
+type WebhookEvent implements Model {
+  "Event id"
+  id: ID!
+  "Type of event"
+  type: String!
+  "Stringified JSON data"
+  data: String!
+}
+
+type WebhookEventsConnection {
```

Comment:
I'm not sure why `Assets` and `Peers` pagination results were called `AssetsConnection` and `PeersConnection` originally... something like `WebhookEventsPaginationResults` make more sense to me IMO

Reply:
My understanding is that it's tied to the way we do pagination: https://relay.dev/graphql/connections.htm in our [baseModel.ts](https://github.com/interledger/rafiki/blob/210b201f8261a46f9df83f2dd649bfa791beb5c9/packages/backend/src/shared/baseModel.ts#LL41C1-L47C6) 

I'm not that familiar with cursor based pagination and the relay spec, so `Connection` was not immediately obvious to me. But it looks like the correct name for the specification we implemented.

Reply:
I see, thanks for the article, now I remember!
There are a few related posts we can consider when adding additional filters:
- https://github.com/graphql/graphql-relay-js/issues/20
- https://medium.com/@conrardy/expanding-relay-cursor-connections-cb6b84fee1e8

Reply:
great resources on this topic, thank you for sharing. I will read these and revisit the `GetWebhookEventsInput` - I suspect there will be some improvements I can make in light of these.

Reply:
This makes me think about keeping the Relay connections convention in using multiple parameters instead of a unifying Input Object (GetWebhookEventsInput), as much as I prefer using object as inputs. What do you think?

Reply:
Generally speaking I'd like to follow the convention if there is one. I only did a cursory scan of those links but the "Expanding Relay Cursor Connections" article shows this for querying which I think makes sense:

```gql
type Query {
    users(
        first: Int,
        after: String,
        filterBy: UserFilterBy
        orderBy: UserOrderBy
    ): UserConnection
}
```

So the pagination args are all seperate top-level args and then filterting and ordering is scoped to different input objects. This also remains consistent with our other paginated endpoints. If we ever added filtering we could just add a filterBy input and not have to refactor the pagination stuff.

Reply:
As of now, query would look like this:

```gql
query WebhookEvents($first: Int, $after: String, $filter: WebhookEventFilter) {
  webhookEvents(first: $first, after: $after, filter: $filter) {
    edges {
      cursor
      node {
        createdAt
        data
        id
        type
      }
    }
    pageInfo { ... }
  }
}
```

with inputs like this (actual filter TBD: https://github.com/interledger/rafiki/pull/1454#discussion_r1202593563):

```json
{
  "first": ,
  "after": ,
  "filter": {
    "type": "some_type"
  }
}
```